operations/spire/Update_Spire_Intermediate_CA_Certificate.md

```diff
@@ -0,0 +1,49 @@
+# Update Spire intermediate CA certificate
+
+Starting with CSM 1.2.5, there is a cronjob in the `vault` namespace named
+`spire-intermediate` that runs once a week. If the Spire intermediate CA
+certificate is set to expire in less than a month when that job runs then it
+will replace the certificate with a new one. If this process fails you can
+take the manual steps listed below in order to update the certificate.
+
+## How to obtain the expiration date for the Spire intermediate CA
+
+To obtain the expiration date of the Spire intermediate CA certificate, run the
+following command on a node that has access to kubectl (Such as ncn-m001)
+
+```bash
+kubectl get secret -n spire spire.spire.ca-tls -o json | jq -r '.data."tls.crt" | @base64d' | openssl x509 -noout -enddate
+```
+
+## How to replace the Spire intermediate CA certificate with a new one
+
+In order to replace the expired or soon to expire Spire intermediate CA
```

Comment:
1. Replace the expired or soon to expire Spire intermediate CA
certificate.

    1. Delete the secret that stores the certificate.
        
        ```bash
        SPIRE_INTERMEDIATE_JOB=$(kubectl get job -n vault -o name| grep 'spire-intermediate' | tail -n1)
        kubectl get secrets -n spire spire.spire.ca-tls -o yaml > spire.spire.ca-tls.yaml.bak
        kubectl delete secret -n spire spire.spire.ca-tls
        ```

    3. Re-run the job that obtains the certificate and creates the secret.

        ```bash
        kubectl get -n vault "$SPIRE_INTERMEDIATE_JOB" -o json | jq 'del(.spec.selector,.spec.template.metadata.labels)' | 
        kubectl replace --force -f -
        ```